bin/system_info/test-print-system-info.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
```

Comment:
trivial/debatable: We are mixing Python and C scripts in the system_info/* directory. Should we move CMakeLists.txt and print_system_info.c into another directory?

Reply:
Another option is moving python script to integration-testing directory like http https://github.com/awslabs/aws-c-http/tree/main/integration-testing.

Reply:
🤷‍♀️ I'm fine mixing file types in a folder.
I personally prefer to group things by "system" vs type.
The theory being:
- if you're deleting the `bin/system_info` directory, you delete everything related to it.
- If you're copying the `bin/system_info` directory, to make another tool like it... hey what's this python script doing in here? an integration test? huh, maybe my new tool should also have an integration test